hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -157,6 +148,25 @@ public void handle(@NonNull final HandleContext handleContext) {
         topicStore.put(builder.build());
     }
 
+    @NonNull
+    @Override
+    public Fees calculateFees(@NonNull final FeeContext feeContext) {
```

Comment:
Just a note, you can check if `TopicCreateSuite`, `TopicUpdateSuite` and `TopicDeleteSuite` `feeAsExpected` are still working.

Reply:
Yes, I checked and they passed.